src/components/pages/EditProject.tsx

```diff
@@ -44,6 +49,7 @@ const EditProject = () => {
 
   if (loading) return <Loading />;
```

Comment:
This shoudl consider whether the project permission query is loading too

Reply:
Actually, this is wrapped in `ProjectEditRoute`. So we shouldn't need to make. project permission query in this component at all.

Reply:
fixed